src/locales/en.json

```diff
@@ -1,10 +1,16 @@
 {
-  "Blank": "Blank",
   "Catalog":"Catalog",
-  "Click the backdrop to dismiss.": "Click the backdrop to dismiss.",
   "Orders":"Orders",
-  "More":"More",
-  "Ready to create an app?": "Ready to create an app?",
-  "Start with Ionic": "Start with Ionic",
-  "UI Components": "UI Components"
+  "OPEN":"OPEN",
```

Comment:
These strings should be sentence case always. Actual case of the label will be handled by the actual component.